docs/bundles/SyliusTaxationBundle/configuring_taxation.rst

```diff
@@ -17,19 +17,19 @@ First step is to create a new tax category.
         $repository = $this->container->get('sylius.repository.tax_category');
         $manager = $this->container->get('sylius.manager.tax_category');
 
-        $clothing = $repository
+        $cloudServerTaxCategory = $repository
             ->createNew()
```

Comment:
@pamil 
I've just used same API used in current docs.
What's the API for factory, so I can change it?